.github/CODEOWNERS

```diff
@@ -4,4 +4,4 @@
 # the repo. Unless a later match takes precedence,
 # these owners will be requested for review when someone
 # opens a pull request.
-*       @dav3r @felddy @jsf9k @mcdonnnj @kcoble @cisagov/team-ois
+*       @dav3r @felddy @hillaryj @jsf9k @mcdonnnj @cisagov/team-ois
```

Comment:
Should we leave Krysta since she's the author?

Reply:
I puzzled over that too.  @kcoble doesn't work here anymore, and I doubt she wants to maintain this repo.

If I'm wrong, @kcoble, just let me know and I'll happily add you again!

Reply:
I miss @kcoble .  We had some fun conversations in the lab!